www/nestedexample/buildbot_nestedexample/__init__.py

```diff
@@ -1,3 +1,6 @@
+import functools
```

Comment:
Seems like unnecessary change

Reply:
Could be an artifact of taking commits apart.

Reply:
fixed.